test/scripts/gather

```diff
@@ -0,0 +1,297 @@
+            container_dir, context, namespace, name, container, previous=True
+        )
+
+    elapsed = time.monotonic() - start
+    logging.debug("Pod %s/%s logs gathered in %.3f seconds", namespace, name, elapsed)
+
+
+def gather_container_logs(
+    base_dir,
+    context,
+    namespace,
+    pod_name,
+    container_name,
+    previous=False,
+):
+    # We don't use drenv.kubectl since it does not support redirection yet.
+    cmd = [
+        "kubectl",
+        "logs",
+        pod_name,
+        f"--namespace={namespace}",
+        f"--container={container_name}",
+        f"--context={context}",
+    ]
+    if previous:
+        cmd.append("--previous")
+    cmd.append(f"--context={context}")
+
+    filename = os.path.join(base_dir, "previous.log" if previous else "current.log")
+    with open(filename, "w") as f:
+        cp = subprocess.run(cmd, stdout=f, stderr=subprocess.PIPE)
+        if cp.returncode != 0:
+            # Log the error, we don't fail the entire operation.
+            logging.debug(
+                "Cannot gather %s logs for container %s/%s/%s: %s",
+                "previous" if previous else "current",
+                namespace,
+                pod_name,
+                container_name,
+                cp.stderr.decode(),
+            )
+
+
+def gather_command(base_dir, context, *cmd):
+    description = " ".join(cmd)
+    logging.debug("Gathering command '%s'", description)
+    start = time.monotonic()
+
+    out = ceph.tool(context, *cmd)
+    filename = os.path.join(base_dir, "-".join(cmd))
+    with open(filename, "w") as f:
+        f.write(out)
+
+    elapsed = time.monotonic() - start
+    logging.debug("Command '%s' gathered in %.3f seconds", description, elapsed)
+
+
+def gather_node(base_dir, context):
+    logging.debug("Gathering node '%s'", context)
+    start = time.monotonic()
+
+    node_dir = create_dir(base_dir, context)
+    gather_rook_ceph_logs(node_dir, context)
+
+    elapsed = time.monotonic() - start
+    logging.debug("Node '%s' gathered in %.3f seconds", context, elapsed)
+
+
+def gather_rook_ceph_logs(base_dir, context):
+    rook_ceph_dir = create_dir(base_dir, "rook-ceph")
+
+    logging.debug("Gathering rbd-mirror client log")
+    source = "/data/rook/rook-ceph/log/ceph-client.rbd-mirror.a.log"
+    target = os.path.join(os.path.abspath(rook_ceph_dir), os.path.basename(source))
+    gather_node_log(context, source, target)
+
+    logging.debug("Looking up rbd mirror peer name")
+    peer_name = kubectl.get(
+        "cephblockpool",
+        "replicapool",
+        "--namespace=rook-ceph",
+        "--output=jsonpath={.status.mirroringInfo.peers[0].site_name}",
+        context=context,
+    )
+
+    logging.debug("Gathering rbd-mirror peer log")
+    source = f"/data/rook/rook-ceph/log/{peer_name}-client.rbd-mirror-peer.log"
+    target = os.path.join(os.path.abspath(rook_ceph_dir), os.path.basename(source))
+    gather_node_log(context, source, target)
```

Comment:
We can simplify and make this future proof by copying all logs files in /data/rook/rook-ceph/log/.

We can discover all logs like this:

```
$ minikube ssh -p dr2 'ls -1 /data/rook/rook-ceph/log/*.log'
/data/rook/rook-ceph/log/b4d98a08-b224-4522-8e9b-14b7847cacbf-client.rbd-mirror-peer.log
/data/rook/rook-ceph/log/ceph-client.ceph-exporter.log
/data/rook/rook-ceph/log/ceph-client.rbd-mirror.a.log
/data/rook/rook-ceph/log/ceph-mgr.a.log
/data/rook/rook-ceph/log/ceph-mon.a.log
/data/rook/rook-ceph/log/ceph-osd.0.log
/data/rook/rook-ceph/log/ceph-volume.log
```

And then copy every log.